lib/middleware/parameter.js

```diff
@@ -4,6 +4,8 @@ const { simplecc } = require('simplecc-wasm');
 const got = require('@/utils/got');
 const config = require('@/config').value;
 const { RE2JS } = require('re2js');
+const marked = require('marked');
```

Comment:
You can use
https://github.com/DIYgod/RSSHub/blob/2c8f4636309977b9d4660781eefbc30317b317a5/package.json#L126
instead of introducing a similar dependency.